RoslynPluginGenerator/UIResources.resx

```diff
@@ -155,6 +155,9 @@
   <data name="NG_DownloadingPackage" xml:space="preserve">
     <value>Downloading package...</value>
   </data>
+  <data name="NG_ERROR_PackageInstallFail" xml:space="preserve">
+    <value>Failed to install package: {0}</value>
```

Comment:
Can you provide more help to users from this error message? You could include some possible causes, including the "pre-release" problem